src/document/utils/index.ts

```diff
@@ -1,2 +1,17 @@
 export * from './base';
+export {
```

Comment:
@acaldas @gpuente vitest has [this weird bug](https://github.com/vitejs/vite/issues/14433) that causes this type of export (`* from './base'`) to break in the tests. Re-exporting them like this fixes the issue without having to change any other code. It seems like they are working on it and we will likely be able to undo this in the future.

If you know of a better way please let me know. What's super annoying is this issue is caused by vitest running the tests in `ssr` mode (described in the issue) but they do not provide any way to opt out of using ssr mode (as far as I can tell from the docs).